server/controllers/request.js

```diff
@@ -0,0 +1,83 @@
+const User = require("../models/User");
+const Profile = require("../models/Profile");
+const Request = require("../models/Request");
+
+exports.createRequest = async (req, res) => {
+    const { ownerId, sitterId, start, end, animalType, usefulInfo } = req.body;
+    console.log(new Date());
+    try {
+        const ownerExists = await User.findOne({ ownerId });
+        if (!ownerExists) return res.status(404).json({ message: "Owner doesn't exist" });
+
+        const sitterExists = await User.findOne({ sitterId });
+        if (!sitterExists) return res.status(404).json({ message: "Sitter doesn't exist" });
+
+        const request = await Request.create(req.body);
+        if (!request) return res.status(404).json({ message: "unable to create a new request" });
+
+        res.status(200).json(request);
+    } catch (error) {
+        res.status(404).json({ message: error.message });
+    }
+};
+
+exports.getRequests = async (req, res) => {
+    try {
+        const userExists = await User.findOne({ _id: req.user.id });
+        if (!userExists) return res.status(404).json({ message: "User doesn't exist" });
+
+        const userProfile = await Profile.findOne({ userId: req.user.id});
+        const typeOfAccount = userProfile.accountType;
+
+        if(typeOfAccount === 'pet_sitter') {
+            const requests = await Request.find({ sitterId: req.user.id }).lean();
+            if (!requests) return res.status(404).json({ message: "Something went wrong!!!" });
```

Comment:
```suggestion
```
A mongoose `find()` query returns an empty list, we can return this to the FE to have a consistent response format.  Also, the above `if` clause will always evaluate to false as an empty list is a truthy value.

Ditto where applicable